buildHook.sh

```diff
@@ -21,34 +21,25 @@ nvm use 14
 
 echo "====== Running build hook ======="
```

Comment:
Dit bestand is niet meer nodig toch?

Reply:
mmmh gebruikte ik om die dist files aan repo toe te voegen:  dist/bootstrap-tabs.js  etc..

Reply:
Maar daar is toch een 'npm run build' oid voor neem ik aan?